tools/CreditsTool/CreditsTool.sln

```diff
@@ -0,0 +1,25 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
```

Comment:
wtf is this?

Reply:
i have no idea it just showed up

Reply:
You used visual studio not visual studio code didn't you

Reply:
> You used visual studio not visual studio code didn't you

![image](https://github.com/Foundation-19/Foundation-19/assets/111151681/30019394-f557-4dbf-a94b-0952eb53f1a2)

Reply:
removed it